lib/bandit/pipeline.ex

```diff
@@ -238,4 +241,15 @@ defmodule Bandit.Pipeline do
       {:ok, transport}
     end
   end
+
+  defp handle_throw(value, stacktrace, transport, span, opts) do
+    Bandit.Telemetry.span_exception(span, :throw, value, stacktrace)
+    status = 500
+
+    if status in Keyword.get(opts.http, :log_exceptions_with_status_codes, 500..599) do
+      Logger.error(Exception.format(:throw, value, stacktrace), domain: [:bandit])
+      Bandit.HTTPTransport.send_on_error(transport, value)
+      {:error, value}
+    end
+  end
```

Comment:
I guess one possibility might be to make `handle_error` more generalized and also handle this `throw` case.

Didn't want to purse that refactor yet for the sake of better code review-ability here. But let me know if you rather prefer that now :-)

Reply:
I don't see any reason why not to handle these in a combined `handle_error` function which takes `:error` or `:throw` as a parameter!

Reply:
Cool. Updated.